course_management/settings.py

```diff
@@ -149,13 +152,12 @@
 # Static files (CSS, JavaScript, Images)
 # https://docs.djangoproject.com/en/4.2/howto/static-files/
 
-STATIC_ROOT = BASE_DIR / "static"
+STATIC_ROOT = BASE_DIR / "staticfiles"
 STATIC_URL = "static/"
 
-# STATICFILES_DIRS = [
-#     STATIC_ROOT,
-#     # Other directories here
-# ]
+STATICFILES_DIRS = [
```

Comment:
why do we need it?

Reply:
I have some issues with tailwind styling when update questions but now is well